Configuration/webapp/app/components/user-section/UserSection.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import React from 'react';
+import { Box, IconButton, Menu, MenuItem, Avatar } from '@mui/material';
+
+interface UserSectionProps {
+  userName: string;
+}
+
+const UserSection: React.FC<UserSectionProps> = ({ userName }) => {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
```

Comment:
As far as I know, it is best practice to import and use only what is needed. Thus, rather than importing all React, you import only `useState`